persistence/soft-index/pom.xml

```diff
@@ -81,6 +81,58 @@
                </instructions>
             </configuration>
          </plugin>
+         <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <configuration>
+               <workingDirectory>${project.build.directory}</workingDirectory>
+            </configuration>
+            <executions>
+               <execution>
+                  <id>defaults</id>
+                  <phase>process-classes</phase>
+                  <goals>
+                     <goal>java</goal>
+                  </goals>
+                  <configuration>
+                     <mainClass>org.infinispan.commons.configuration.defaults.DefaultAttributeExtractor</mainClass>
```

Comment:
@slaskawi Yeah I did think that a Maven plugin would maybe better once I started pasting the same stuff into the pom. I've never developed a plugin before, so I'll look into what that entails etc